packages/data-point/lib/entity-types/integrations.test.js

```diff
@@ -268,3 +268,39 @@ describe('trace feature', () => {
       })
   })
 })
+
```

Comment:
these tests are mostly here for documentation purposes, the same test is done on each entity, might seem redundant but has been helpful for me and others